src/sentry/api/serializers/models/rule.py

```diff
@@ -43,19 +51,23 @@ def get_attrs(self, item_list, user, **kwargs):
 
     def serialize(self, obj, attrs, user):
         environment = attrs["environment"]
+        all_conditions = [
```

Comment:
because all filters still live in rule.data["conditions"] as to not disrupt the rule json schema too much we must separate them out before sending to the frontend

all_conditions -> conditions, filters